CONTRIBUTING.rst

```diff
@@ -95,48 +95,48 @@ managing your python evironment using your operating system's package manager or
     $ export PYENV_ROOT="$HOME/.pyenv"
     $ export PATH="$HOME/.pyenv/bin:$PATH
     $ eval "$(pyenv init -)"
-    $ pyenv install -s 3.6.14
-    $ pyenv install -s 3.7.11
-    $ pyenv install -s 3.8.11
-    $ pyenv install -s 3.9.6
+    $ pyenv install -s 3.7.12
```

Comment:
We don't need this line anymore, correct?

Reply:
good call. Cleaned it as well as other remaining occurrences